src/SDKs/Maps/Management.Maps/Microsoft.Azure.Management.Maps.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.Maps</PackageId>
+    <PackageReleaseNotes>This package replaces Microsoft.Azure.Management.LocationBasedServices. Azure Location Based Services is now Azure Maps.</PackageReleaseNotes>
```

Comment:
"Open a PR for the Microsoft.Azure.Management.Maps csproj, indicate that this is a replacement for the Microsoft.Azure.Management.LocationBasedServices in the PackageReleaseNotes and release the package"